test/tests/functional/pbs_svr_dyn_res.py

```diff
@@ -615,17 +615,17 @@ def test_svr_dyn_res_permissions(self):
 
         # give write permission to user only
         self.du.chmod(path=fp, mode=0o744, sudo=True)
-        if os.getuid() != 0:
-            self.check_access_log(fp, exist=True)
-        else:
-            self.check_access_log(fp, exist=False)
+        self.check_access_log(fp, exist=False)
```

Comment:
What about the case where daemon user  is root and os.getuid() is not?

Reply:
we don't call os.getuid(), i don't understand

Reply:
Look at the code you replaced.  It was an if statement with os.getuid() != NULL to call check_access_log() with exist=True.  Otherwise call it with exist=False.  You just dumped the first part of that if statement

Reply:
i dumped it because we only cared about os.getuid() when we used the current user's home directory. 
Since i changed it to /tmp,